src/types.ts

```diff
@@ -1,42 +1,4 @@
 export type Color =
-  | 'white'
-  | 'black'
-  | 'skyLighter'
-  | 'skyLight'
-  | 'sky'
-  | 'skyDark'
-  | 'inkLightest'
-  | 'inkLighter'
-  | 'inkLight'
-  | 'ink'
-  | 'blueLighter'
-  | 'blueLight'
-  | 'blue'
-  | 'blueDark'
-  | 'blueDarker'
-  | 'indigoLighter'
-  | 'indigoLight'
-  | 'indigo'
-  | 'indigoDark'
-  | 'indigoDarker'
-  | 'tealLighter'
-  | 'tealLight'
-  | 'teal'
-  | 'tealDark'
-  | 'tealDarker'
-  | 'greenLighter'
-  | 'green'
-  | 'greenDark'
-  | 'yellowLighter'
-  | 'yellow'
-  | 'yellowDark'
-  | 'orange'
-  | 'redLighter'
-  | 'red'
-  | 'redDark'
-  | 'purple';
-
-export type NewDesignLanguageColor =
```

Comment:
For weird legacy never-got-around-to-fixing-it reasons everything in this file is exported publicly. So `Color` remains exported but now has a very different shape. Will that more problems than "Color is no longer exported" from this file?

I'm thinking it might, and it'd be easier to keep this named as `NewDesignLanguageColor`.

Reply:
Hey @BPScott I don't want to forget about this. As this is a breaking change I think we should be deprecating `NewDesignLanguageColor`. Should we also make the `Color` export private so it cannot be used externally??